concepticondata/conceptlists/DellaRosa-2010-417.tsv

```diff
@@ -0,0 +1,418 @@
+ID	NUMBER	ITALIAN	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS	ITALIAN_CONCRETENESS_MEAN	ITALIAN_IMAGEABILITY_MEAN	ITALIAN_FAMILIARITY_MEAN	ITALIAN_AoA_MEAN	ITALIAN_CONTEXT_AVAILABILITY_MEAN	ITALIAN_ABSTRACTNESS_MEAN	ITALIAN_MoA_MEAN
+DellaRosa-2010-417-1	1	Abbandono	abandonment			297	300	397	374	413	455	505
+DellaRosa-2010-417-2	2	Accelerazione	acceleration			344	342	468	445	409	415	488
+DellaRosa-2010-417-3	3	Accordo	deal			262	300	468	388	368	459	547
+DellaRosa-2010-417-4	4	Affermazione	affirmation			274	265	516	424	376	375	425
+DellaRosa-2010-417-5	5	Affetto	affection	3834	LOVE (AFFECTION)	284	465	584	294	550	488	363
+DellaRosa-2010-417-6	6	Agitazione	perturbation			413	311	541	320	497	441	364
+DellaRosa-2010-417-7	7	Agonia	agony	1783	PAIN	294	332	305	480	444	476	600
```

Comment:
We're not consistent with the agony/pain mappings in Concepticon and I'm not entirely happy with mapping agony to pain. What do you think?

Reply:
I agree and would unmap.